x-pack/plugins/cases/server/routes/api/cases/sub_case/find_sub_cases.ts

```diff
@@ -45,50 +38,13 @@ export function initFindSubCasesApi({ caseService, router, logger }: RouteDeps)
           fold(throwErrors(Boom.badRequest), identity)
         );
 
-        const ids = [request.params.case_id];
-        const { subCase: subCaseQueryOptions } = constructQueryOptions({
-          status: queryParams.status,
-          sortByField: queryParams.sortField,
-        });
-
-        const subCases = await caseService.findSubCasesGroupByCase({
-          soClient,
-          ids,
-          options: {
-            sortField: 'created_at',
-            page: defaultPage,
-            perPage: defaultPerPage,
-            ...queryParams,
-            ...subCaseQueryOptions,
-          },
-        });
-
-        const [open, inProgress, closed] = await Promise.all([
-          ...caseStatuses.map((status) => {
-            const { subCase: statusQueryOptions } = constructQueryOptions({
-              status,
-              sortByField: queryParams.sortField,
-            });
-            return caseService.findSubCaseStatusStats({
-              soClient,
-              options: statusQueryOptions ?? {},
-              ids,
-            });
-          }),
-        ]);
-
+        const client = await context.cases.getCasesClient();
         return response.ok({
-          body: SubCasesFindResponseRt.encode(
-            transformSubCases({
-              page: subCases.page,
-              perPage: subCases.perPage,
-              total: subCases.total,
-              subCasesMap: subCases.subCasesMap,
-              open,
-              inProgress,
-              closed,
-            })
-          ),
+          body: await client.subCases.find({
+            soClient,
```

Comment:
Same as before `soClient` shouldn't be part of the arguments. The `CasesClient` provide it to all sub clients.